test/test_prototype_transforms_functional.py

```diff
@@ -860,3 +866,26 @@ def test_correctness_crop_bounding_box(device, top, left, height, width, expecte
     )
 
     torch.testing.assert_close(output_boxes.tolist(), expected_bboxes)
+
+
+@pytest.mark.parametrize("device", cpu_and_gpu())
+def test_correctness_vertical_flip_segmentation_mask_on_fixed_input(device):
+    mask = torch.tensor(
```

Comment:
Personally I'd prefer to create them as torch zeros and assign specific values to one, if possible. Also reduce the number of lines and improve readability. What do you think @federicopozzi33

Reply:
I think it's a good idea.

I was creating a 5x5x3 just for having a nice indentation, and increasing readibility. But, IMO 3x3x3 is enough.

I also set the dtype, which was missing.